dotcom-rendering/src/web/components/KeyEventCard.stories.tsx

```diff
@@ -37,39 +37,69 @@ const wrapperStyles = css`
 	}
 `;
 
-const Cards = ({ theme, count }: { theme: ArticleTheme; count: number }) => (
+const SummaryCard = ({ theme }: { theme: ArticleTheme }) => (
+	<ul css={wrapperStyles}>
+		<KeyEventCard
+			text={events[0].text}
+			url={events[0].url}
+			date={events[0].date}
+			format={getFormat(theme)}
+			isSummary={true}
+		/>
+	</ul>
+);
+
+const StandardCard = ({
+	theme,
+	count,
+}: {
+	theme: ArticleTheme;
+	count: number;
+}) => (
 	<ul css={wrapperStyles}>
 		{events.slice(0, count).map((event) => (
 			<KeyEventCard
 				text={event.text}
 				url={event.url}
 				date={event.date}
 				format={getFormat(theme)}
+				isSummary={event.isSummary}
 			/>
 		))}
 	</ul>
 );
 
-const SingleCard = () => (
+const Summary = () => (
+	<>
+		<SummaryCard theme={ArticlePillar.News} />
+		<SummaryCard theme={ArticlePillar.Culture} />
+		<SummaryCard theme={ArticlePillar.Lifestyle} />
+		<SummaryCard theme={ArticlePillar.Sport} />
+		<SummaryCard theme={ArticlePillar.Opinion} />
+		<SummaryCard theme={ArticleSpecial.SpecialReport} />
+	</>
+);
+
+const KeyEvent = () => (
 	<>
-		<Cards theme={ArticlePillar.News} count={1} />
-		<Cards theme={ArticlePillar.Culture} count={1} />
-		<Cards theme={ArticlePillar.Lifestyle} count={1} />
-		<Cards theme={ArticlePillar.Sport} count={1} />
-		<Cards theme={ArticlePillar.Opinion} count={1} />
-		<Cards theme={ArticleSpecial.SpecialReport} count={1} />
+		<StandardCard theme={ArticlePillar.News} count={1} />
+		<StandardCard theme={ArticlePillar.Culture} count={1} />
+		<StandardCard theme={ArticlePillar.Lifestyle} count={1} />
+		<StandardCard theme={ArticlePillar.Sport} count={1} />
+		<StandardCard theme={ArticlePillar.Opinion} count={1} />
+		<StandardCard theme={ArticleSpecial.SpecialReport} count={1} />
 	</>
 );
 
-const MultipleCards = () => (
+const Multiple = () => (
```

Comment:
Would it be worth adding a summary card into this?

Reply:
The `events` data includes a `Summary` event so this is covered here - agree though its not obvious from the naming.